textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/ListField.java

```diff
@@ -38,7 +38,7 @@ public boolean equals(Object obj) {
         if (list == null) {
             if (other.list != null)
                 return false;
-        } else if (!list.equals(other.list))
+        } else if (!list.containsAll(other.list))
```

Comment:
Is it because `equals()` cares about order, why we don't?  If so, then the name "list" is not accurate.  Should we rename it to "SetField"?